src/app/login/Login.styles.ts

```diff
@@ -0,0 +1,10 @@
+export const styles = {
+  wrapper: {
+    marginTop: 8,
```

Comment:
spacing

Reply:
Tutaj i w innych miejscach gdzie to jest użyte  `{ marginTop: 8 } === { marginTop: theme.spacing(8) }`. To tylko taki "syntactic sugar". Czy w związku z tym i tak powinno to być zrobione przez `theme.spacing()`?

Reply:
Nie, sorry, nie zauważyłem że to już jest przetworzone ;)